src/Tablebot/Plugin/Parser.hs

```diff
@@ -82,6 +84,34 @@ discordUser = do
   num <- between (chunk "<@") (single '>') (some digit)
   return $ "<@" ++ num ++ ">"
 
+-- | @NrQuery@ stores the string within a Netrunner query with its query type.
```

Comment:
Not sure this stuff ultimately wants to be in Plugin/Parser, but moving that can be part of the restructure I mentioned in the chat rather than now